File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    arr_x=[]
    arr_y=[]
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[i])):
            if wallpaper[i][j] == "#":
                arr_x.append(i)
                arr_y.append(j)
    print(min(arr_x),min(arr_y),max(arr_x)+1,max(arr_y)+1)
    answer = [min(arr_x),min(arr_y),max(arr_x)+1,max(arr_y)+1]
    return answer

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
import java.util.*;
class Solution {
    public int[] solution(String[] wallpaper) {
        ArrayList<Integer> xArr = new ArrayList<>();
        ArrayList<Integer> yArr = new ArrayList<>();
        for(int i=0; i<wallpaper.length; i++) {
        	for(int j=0; j<wallpaper[i].length();j++) {
        		String data = wallpaper[i].charAt(j)+"";
        		if(data.equals("#")) {
        			xArr.add(i);
        			yArr.add(j);
        		}
        	}
        }
        xArr.sort((a,b)->{return a-b;});
        yArr.sort((a,b)->{return a-b;});
        int[] answer = {xArr.get(0),yArr.get(0),xArr.get(xArr.size()-1)+1,yArr.get(yArr.size()-1)+1};
        return answer;
    }
}